doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -72,4 +72,11 @@ Access GeoServer front page
 ---------------------------
 
 * After a few seconds, GeoServer should be accessible at: `<http://localhost:8080/geoserver>`_
-* The default ``admin`` password is ``geoserver``.
\ No newline at end of file
+* The default ``admin`` password is ``geoserver``.
+
+Run GeoServer from Intellij on Windows
+---------------------------
+
+#. Add bash to your Windows environment path and restart Intellij.  
```

Comment:
I set up once on a Windows machine without seeing this... but it likely had git bash in the path already, first thing I do when on Windows is to make it a bit "linux like" :-p